config/sync/views.view.pages_with_no_theme.yml

```diff
@@ -0,0 +1,854 @@
+              empty_column: false
+              responsive: ''
+          override: true
+          sticky: false
+          summary: ''
+          empty_table: false
+          caption: ''
+          description: ''
+      row:
+        type: 'entity:node'
+        options:
+          view_mode: teaser
+      query:
+        type: views_query
+        options:
+          query_comment: ''
+          disable_sql_rewrite: false
+          distinct: false
+          replica: false
+          query_tags: {  }
+          contextual_filters_or: false
+      relationships:
+        field_subtheme:
+          id: field_subtheme
+          table: node__field_subtheme
+          field: field_subtheme
+          relationship: none
+          group_type: group
+          admin_label: 'field_subtheme: Taxonomy term'
+          plugin_id: standard
+          required: false
+        nid:
+          id: nid
+          table: node_field_revision
+          field: nid
+          relationship: none
+          group_type: group
+          admin_label: 'Get the actual content from a content revision.'
+          entity_type: node
+          entity_field: nid
+          plugin_id: standard
+          required: false
+        revision_uid:
+          id: revision_uid
+          table: node_revision
+          field: revision_uid
+          relationship: none
+          group_type: group
+          admin_label: 'revision user'
+          entity_type: node
+          entity_field: revision_uid
+          plugin_id: standard
+          required: false
+      header:
+        area:
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: text
+          empty: true
+          content:
+            value: '<p>Pages listed below are missing a theme/subtheme.  This usually means a theme that the page was tagged with has been deleted.  For some page types (such as webforms), a theme is optional and the lack of a theme may be intentional. For other page types listed here, if the page is published, the page should be given a theme to ensure links to the page are displayed in the correct theme and to ensure that the page breadcrumb trail and related content menus work correctly.</p>'
+            format: basic_html
+          tokenize: false
+        result:
+          id: result
+          table: views
+          field: result
+          relationship: none
+          group_type: group
+          admin_label: ''
+          plugin_id: result
+          empty: false
+          content: 'Displaying @start - @end of @total'
+      footer: {  }
+      display_extenders: {  }
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - url.query_args
+        - user
+        - 'user.node_grants:view'
+        - user.permissions
+      tags: {  }
+  page_1:
```

Comment:
Where possible, it's best to set these display ids to something more descriptive. Helps immensely with debugging and general readability